src/java.base/share/classes/java/util/Locale.java

```diff
@@ -327,19 +322,87 @@
  * category. In the absence of category specific system properties, the "category-less"
  * system properties are used, such as {@code user.language} in the previous example.
  *
- * <h3><a id="LocaleMatching">Locale Matching</a></h3>
+ * <h2><a id="ObtainingLocale">Obtaining a Locale</a></h2>
  *
- * <p>If an application or a system is internationalized and provides localized
+ * <p>There are several ways to obtain a {@code Locale} object.
+ * It is advised against using the deprecated {@code Locale} constructors.
+ *
+ * <dl>
+ *  <dt><b>Locale Constants</b></dt>
+ *  <dd>A number of convenient constants are provided that return {@code Locale}
+ *  objects for commonly used locales. For example, {@link #US Locale.US} is the
+ *  {@code Locale} object for the United States.</dd>
+ *  <dt><b>Factory Methods</b></dt>
+ *  <dd>{@link #of(String, String, String) Locale::of} and its overloads obtain a
+ *  {@code Locale} object from the given {@code language}, {@code country},
+ *  and/or {@code variant}. {@link #forLanguageTag(String)} obtains a {@code Locale}
+ *  object for a well-formed BCP 47 language tag.</dd>
+ *  <dt><b>Builder</b></dt>
+ *  <dd>{@link Builder} is used to construct a {@code Locale} object that conforms
+ *  to BCP 47 syntax. Use a builder to enforce syntactic restrictions on the input.</dd>
+ * </dl>
+ * <p>The following are all equivalent:
```

Comment:
"all equivalent" seems a bit vague, as the actual implementation is not equivalent (unlike other examples below). Something like "the following all produce equivalent locale objects" or along the lines?